<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-KAQJQP02" ignored="0" name="D:/2022/Java Projects/Capstone_project_Test_Automation" tests="2" failures="1" timestamp="2022-12-20T16:31:33 PST" time="26.329" errors="0">
  <testcase name="testFinishPurchase" time="0.134" classname="tests.SR_12130">
    <failure type="java.lang.AssertionError" message="Fail purchase, expected messageThank you for your purchase!, got: Purchase fail! expected [Thank you for your purchase!] but found [Purchase fail!]">
      <![CDATA[java.lang.AssertionError: Fail purchase, expected messageThank you for your purchase!, got: Purchase fail! expected [Thank you for your purchase!] but found [Purchase fail!]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at tests.CustomAssertions.isSuccessPurchase(CustomAssertions.java:45)
at tests.SR_12130.testFinishPurchase(SR_12130.java:42)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:750)
]]>
    </failure>
  </testcase> <!-- testFinishPurchase -->
  <testcase name="testListCart" time="25.16" classname="tests.SR_12130"/>
</testsuite> <!-- D:/2022/Java Projects/Capstone_project_Test_Automation -->
